src/gui/editors/PianoRoll.cpp

```diff
@@ -1850,53 +1850,49 @@ void PianoRoll::computeSelectedNotes(bool shift)
 
 
 
-void PianoRoll::mouseReleaseEvent(QMouseEvent * me )
+void PianoRoll::mouseReleaseEvent( QMouseEvent * me )
 {
-	s_textFloat->hide();
 	bool mustRepaint = false;
 
+	s_textFloat->hide();
+
 	if( me->button() & Qt::LeftButton )
 	{
 		m_mouseDownLeft = false;
 		mustRepaint = true;
-	}
-	if( me->button() & Qt::RightButton )
-	{
-		m_mouseDownRight = false;
-		mustRepaint = true;
```

Comment:
Did you check if these two assignments being moved below the usage of other internal functions did not break anything?

Because this could be here in this awkward position for a good reason.